### PR TITLE
Fix grammar in client function comments

### PR DESCRIPTION
**Pull request by LeviHarrison**
**- What I did**

I was looking through the client documentation and found a small grammar mistake in one of the comments, and ended up falling down this rabbit hole. I change certain words and added punctuation to the comments of functions. I didn't get everything, nor is my grammar perfect, and I'd be happy to move things around.

**- Description for the changelog**

Grammar fixes in exported client functions

**- A picture of a cute animal (not mandatory but encouraged)**

image

### CODE REVIEW

File: client/image_build.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/docker/docker/api/types/container"
 )
 
-// ImageBuild sends request to the daemon to build images.
-// The Body in the response implement an io.ReadCloser and it's up to the caller to
+// ImageBuild sends a request to the daemon to build images.
+// The body in the response implements an io.ReadCloser and it's up to the caller to
```
Comment (thaJeztah) [inline client/image_build.go:18]: I think in this case, `Body` refers to the `Body` field in the response; https://github.com/moby/moby/blob/46cdcd206c56172b95ba5c77b827a722dab426c5/client/image_build.go#L42-L45

**Review by thaJeztah — CHANGES_REQUESTED** [review]
thanks! left some comments

**Comment by LeviHarrison** [comment]
Ok I removed all code changes that were just adding a period (I did leave one in that removed a double period) and recapitalized `Body`.

**Comment by thaJeztah** [comment]
Thanks! Could you perhaps squash the commits? (let me know if you need help/instructions doing so 🤗)

**Comment by LeviHarrison** [comment]
A squash has (sort of) been performed. Ignore all the force pushes...

**Review by thaJeztah — APPROVED** [review]
awesome!

LGTM

**Review by tianon — APPROVED** [review]
LGTM :+1:

**Pull request merged**